crossdock/client/client.go

```diff
@@ -33,7 +34,7 @@ import (
 // Start begins a blocking Crossdock client
 func Start() {
 	http.HandleFunc("/", behaviorRequestHandler)
-	http.ListenAndServe(":8080", nil)
+	log.Fatal(http.ListenAndServe(":8080", nil))
```

Comment:
@kriskowal `Fatal` just prints the given error/object and stops exception. `ListenAndServe` *always* returns an error because it blocks indefinitely, so when it stops is when the listener was stopped and it failed to receive a connection.